kubectl-cloudflow/go.sum

```diff
@@ -151,6 +153,8 @@ github.com/prometheus/common v0.2.0/go.mod h1:TNfzLD0ON7rHzMJeJkieUDPYmFC7Snx/y8
 github.com/prometheus/procfs v0.0.0-20180725123919-05ee40e3a273/go.mod h1:c3At6R/oaqEKCNdg8wHV1ftS6bRYblBhIjjI8uT2IGk=
 github.com/prometheus/procfs v0.0.0-20181005140218-185b4288413d/go.mod h1:c3At6R/oaqEKCNdg8wHV1ftS6bRYblBhIjjI8uT2IGk=
 github.com/prometheus/procfs v0.0.0-20190117184657-bf6a532e95b1/go.mod h1:c3At6R/oaqEKCNdg8wHV1ftS6bRYblBhIjjI8uT2IGk=
+github.com/rayroestenburg/configuration v0.0.0-20200115015912-550403a6bd87 h1:JTGDqHa23LTUjEsJQorToCYedKxI6SM9SkhooYyKlTw=
```

Comment:
pls rm your repo references.

Reply:
This is auto-generated

Reply:
I dont have it on my setup and I can use that lib as is.